bokeh/models/annotations.py

```diff
@@ -1002,7 +1002,7 @@ class Tooltip(Annotation):
     """)
 
     show_arrow = Bool(default=True, help="""
-    Whether tooltip's arrow should be showed.
+    Whether tooltip's arrow should be shown.
```

Comment:
As far I can tell (not a native speaker), both are correct, just, I suppose, "shown" is the preferred one.